com.dynamo.cr/com.dynamo.cr.bob.test/src/com/dynamo/bob/pipeline/LuaScannerTest.java

```diff
@@ -234,7 +234,7 @@ public void testPropsVec4() throws Exception {
     public void testPropsQuat() throws Exception {
         List<Property> properties = getPropertiesFromFile("test_props_quat.lua");
 
-        assertEquals(2, properties.size());
+        assertEquals(3, properties.size());
```

Comment:
Currently because of using RegExp we just ignore such cases and just bundle without errors :

>go.property("prop3", vmath.quat(1a, 2, 3, 4))

Using ANTLR - user will see the error
```
Only these types are available: number, hash, msg.url, vmath.vector3, vmath.vector4, vmath.quat, resource.*
```

So, the property is here, but with an error status, that's why I've fixed test  here.